justspacesapp/static/css/custom.css

```diff
@@ -0,0 +1,268 @@
+:root {
+  --sidebar-background: #fafafa;
+  --sidebar-text: black;
+  --sidebar-text-hover:  black;
+}
+
+body {
+    background: #fafafa;
+    padding-top: 0px;
+}
+
+p {
+    font-size: 1.1em;
+    font-weight: 300;
+    line-height: 1.7em;
+    color: #999;
+}
+
+a,
+a:hover,
+a:focus {
+    color: inherit;
+    text-decoration: none;
+    transition: all 0.3s;
+}
+
+.navbar {
+    padding: 15px 10px;
+    background: #fff;
+    border: none;
+    border-radius: 0;
+    margin-bottom: 40px;
+    box-shadow: 1px 1px 3px rgba(0, 0, 0, 0.1);
+}
+
+.navbar-btn {
+    box-shadow: none;
+    outline: none !important;
+    border: none;
+}
+
+.line {
+    width: 100%;
+    height: 1px;
+    border-bottom: 1px dashed #ddd;
+    margin: 40px 0;
+}
+
+.list-unstyled {
+  padding-left: 30px;
+}
+
+i,
+span {
+    display: inline-block;
+}
+
+/* ---------------------------------------------------
+    SIDEBAR STYLE
+----------------------------------------------------- */
+
+.wrapper {
+    display: flex;
+    align-items: stretch;
+}
+
+#sidebar {
```

Comment:
It's typically best practice to use classes for CSS styles. (You can find myriad discussions about this: a cursory search turned up [this article](https://dev.to/claireparker/reasons-not-to-use-ids-in-css-4ni4).) 

I'll leave it up to you, ultimately, but let me know what you think.

[N.b., I see that you use [`#sidebar` in the JS.](https://github.com/datamade/just-spaces/pull/17/files#diff-a9ed6d5e5bde2db9db7ce31a5712bda4R71) That makes sense.]

Reply:
This was a good read, thanks! I'll switch them over.